skill.json

```diff
@@ -48,8 +48,8 @@
         "- [SoundBible](http://soundbible.com/suggest.php?q=laugh&x=0&y=0)",
         "- [FreeSound](https://freesound.org/search/?q=female+evil+laugh)"
     ],
-    "skillname": "ovos-skill-laugh",
-    "authorname": "OpenVoiceOS",
+    "skillname": "skill-laugh",
```

Comment:
Is this an intentional name change? It will be a breaking change for config/settings

Reply:
No, it's based on the forked repo's name. Same deal as above